scene/main/node.cpp

```diff
@@ -2865,7 +2865,7 @@ void Node::remap_nested_resources(Ref<Resource> p_resource, const HashMap<Ref<Re
 void Node::_duplicate_properties(const Node *p_root, const Node *p_original, Node *p_copy, int p_flags) const {
 	List<PropertyInfo> props;
 	p_original->get_property_list(&props);
-	StringName script_property_name = CoreStringNames::get_singleton()->_script;
+	StringName script_property_name = CoreStringName(script);
```

Comment:
Should it be `const T&`?

There's also an unused one in `_duplicate` that could be removed.

Reply:
Will add!